src/main/resources/testchipip/csrc/mm.cc

```diff
@@ -38,19 +43,36 @@ void mm_t::init(size_t sz, int wsz, int lsz)
   assert(wsz > 0 && lsz > 0 && (lsz & (lsz-1)) == 0 && lsz % wsz == 0);
   word_size = wsz;
   line_size = lsz;
+  seed = std::chrono::system_clock::now().time_since_epoch().count();
```

Comment:
Lets make this seed a static constant. Otherwise behavior will change across compilations of the same config, which complicates debugging